docker/tests/test_pytorch_docker.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
?

Reply:
oh whoops didn't mean to commit this file